cob_moveit_bringup/launch/stomp_moveit.rviz

```diff
@@ -0,0 +1,329 @@
+Panels:
```

Comment:
remove this file...you need to load the proper config locally...it's a duplicate anyway...